src/clients/firstboot.rb

```diff
@@ -42,10 +42,9 @@ def main
 
       Wizard.OpenNextBackStepsDialog
 
-      @wizard_mode = Mode.test ? "installation" : Mode.mode
-
-      @stage_mode = [{ "stage" => "firstboot", "mode" => @wizard_mode }]
-      ProductControl.AddWizardSteps(@stage_mode)
+      # Always force the mode (bsc#924278)
+      Mode.SetMode("installation")
```

Comment:
No, this is a Firstboot client: starts when Yast starts, stops when Yast stops

See also https://github.com/yast/yast-yast2/blob/93f5a6288b8d339b6f13becd1d14d27cba577f94/library/general/src/modules/Mode.rb#L130-L132